sir/amqp/message.py

```
@@ -56,7 +56,7 @@ def from_amqp_message(cls, queue_name, amqp_message):
 
         try:
             data = ujson.loads(amqp_message.body)
-        except StandardError as e:
+        except Exception as e:
```

It is generally a bad idea to try and blanket catch all exceptions like this. In this case it would be OK, but it is is best to catch ValueError, which is thrown in case of unparsable JSON.

---

Fixed with https://github.com/metabrainz/sir/pull/110/commits/66229997f67e9f65cbc87be85b31b46c1516bb3e